Apps/Sandcastle/gallery/3D Tiles Interactivity.html

```diff
@@ -45,7 +45,6 @@
     'use strict';
 //Sandcastle_Begin
 var viewer = new Cesium.Viewer('cesiumContainer');
-
```

Comment:
I'm sorry, @esraerik, my mistake, I meant remove this extra whitespace change. It's not needed.